src/components/pages/Breakdown.vue

```diff
@@ -1095,7 +1114,15 @@ export default {
           entityId,
           casting: castingToPaste
         })
-        this.saveCasting(entityId).then(this.setLock).catch(console.error)
+        this.saveCasting(entityId)
+          .then(() => {
+            delete this.saveErrors[entityId]
```

Comment:
Is it intentional to delete the error only after the promise resolution, rather than when calling the promise as below?

eg.
```js
          delete this.saveErrors[entityId]
          this.saveCasting(entityId)
            .then(() => {
              this.setLock()
            })
```

Reply:
Yes, it's in case you save again

Reply:
I have changed that